08-rails/active-record-models.md

```diff
@@ -192,69 +188,68 @@ SQL (0.3ms)  UPDATE "students" SET "pie" = ?, "motto" = ?, "updated_at" = ? WHER
-| `libby.name`          | Get the value of one field                 | N/A       | Value   | No          |
-| `libby.id`            | Get the unique ID assigned by ActiveRecord | N/A       | Value   | No          |
-| `libby.pie = 'peach'` | Modify the value of one field              | Value     | Value   | No          |
+| `my_book.attributes`    | Get a hash containing all fields           | N/A       | Hash    | No          |
+| `my_book.title`          | Get the value of one field                 | N/A       | Value   | No          |
+| `my_book.id`            | Get the unique ID assigned by ActiveRecord | N/A       | Value   | No          |
+| `my_book.author = 'coool'` | Modify the value of one field              | Value     | Value   | No          |
 
 ### Modifying the DB
-`save`, `create`, `update`, and `destroy` are the only ways to change what's stored in the database - other methods just change local copies of the data. To use the vocabulary we learned for HTTP verbs, these methods are _unsafe_.
+`save`, `create`, `update`, and `destroy` are the only ways to change what's stored in the database - other methods just change **local copies** of the data. To use the vocabulary we learned for HTTP verbs, these methods are _unsafe_.
 
 | Method                                        | Description                                    | Arguments | Returns     | Talks to DB |
 |:----------------------------------------------|:-----------------------------------------------|:----------|:------------|:------------|
-| `libby = Student.new`                         | Make a new entry, stored in a local variable   | Hash      | One Student | No          |
-| `libby.save`                                  | Save a local copy of an entry to the DB        | N/A       | Boolean     | Yes         |
-| `Student.create(name: 'libby', pie: 'peach')` | Make a new a entry and save to the DB          | Hash      | One Student | Yes         |
-| `libby.update(pie: apple)`                    | Modify an entry and save to the DB             | Hash      | One Student | Yes         |
-| `libby.destroy`                               | Remove an entry from the DB, freeze local copy | N/A       | One Student | Yes         |
+| `book = Book.new`                         | Make a new entry, stored in a local variable   | Hash      | One Book | No          |
+| `book.save`                                  | Save a local copy of an entry to the DB        | N/A       | Boolean     | Yes         |
+| `Book.create(author: 'author', title: 'title')` | Make a new a entry and save to the DB          | Hash      | One Book | Yes         |
+| `book.update(author: 'new author')`                    | Modify an entry and save to the DB             | Hash      | One Book | Yes         |
+| `book.destroy`                               | Remove an entry from the DB, freeze local copy | N/A       | One Book | Yes         |
```

Comment:
Below this on line 259 we mention covering the creation of AR models and migrations, however that has since been moved out to a different file.

We should also remove the Rails Guide link for migrations (and maybe ActiveModel basics) in the resources list.